package.json

```
@@ -11,7 +11,7 @@
     "url": "git+https://github.com/gnosis/dex-js.git"
   },
   "scripts": {
-    "build": "rimraf build && ttsc",
+    "build": "rimraf build && ttsc && cp -r src/contracts/gen/ build/contracts/",
```

It's copying the contracts in the src/contracts. It's working if we drop the slash in the source dir:

```suggestion
    "build": "rimraf build && ttsc && cp -r src/contracts/gen build/contracts/",
```